scripts/check_cmake_files.py

```diff
@@ -23,30 +31,43 @@
 import subprocess
 import sys
 
+from pprint import pprint as pp
 
-DEFAULT_DIRS = [
+
+# Directories relative to the repo root that will be scanned by default if no
+# arguments are passed
```

Comment:
Ultranit: add a full stop at the end of the sentence.

Reply:
Done.